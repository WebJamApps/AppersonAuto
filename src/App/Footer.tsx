import React from 'react';

const footerLinks = (): JSX.Element => (
  <>
    <div style={{ textAlign: 'center', padding: '6px' }} className="footerLinks">
      <a href="https://www.facebook.com/pages/Apperson-Automotive/361511727202422">
        <i style={{ color: '#45c9ff' }} className="fab fa-facebook-square fa-2x" aria-hidden="true" />
      </a>
      <a href="https://www.yellowpages.com/salem-va/mip/apperson-automotive-482421530" style={{ marginLeft: '40px' }}>
        <img
          style={{ marginBottom: '15px', verticalAlign: 'middle' }}
          src="https://insightmg.com/wp-content/uploads/2015/12/YP_Insight-Marketing-Group_Marketing-for-Medical-Practices.png"
          alt="yellowpages"
          width="40px"
        />
      </a>
      <a href="https://www.yelp.com/biz/apperson-automotive-salem" style={{ marginLeft: '30px' }}>
        <img
          style={{ marginBottom: '15px', verticalAlign: 'middle' }}
          src="https://dl.dropboxusercontent.com/s/pb5qn5wyps6k56t/yelp-logo-icon-7.jpg?dl=0"
          alt="yelp"
          width="40px"
        />
      </a>
    </div>
    <div style={{ textAlign: 'center', padding: '6px' }}>
      <span className="nohover">
        Powered by &nbsp;
      </span>
      <a className="wjllc" target="_blank" rel="noopener noreferrer" href="https://www.web-jam.com">Web Jam LLC</a>
    </div>
  </>
);

const Footer = (): JSX.Element => (
  <div
    id="wjfooter"
    className="footer"
    style={{
      marginTop: '20px', paddingTop: '20px', paddingBottom: '20px', bottom: '0',
    }}
  >
    { footerLinks() }
  </div>
);

export default Footer;
