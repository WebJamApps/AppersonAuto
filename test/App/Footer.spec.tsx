import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../src/App/Footer';

describe('Footer', () => {
  const wrapper = shallow(<Footer />);
  it('renders footer links', () => {
    expect(wrapper.find('div.footerLinks').exists()).toBe(true);
  });
  it('renders footer', () => {
    expect(wrapper.find('div#wjfooter').exists()).toBe(true);
  });
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
