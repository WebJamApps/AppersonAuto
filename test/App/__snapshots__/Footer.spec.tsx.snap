// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders component 1`] = `
<div
  className="footer"
  id="wjfooter"
  style={
    {
      "bottom": "0",
      "marginTop": "20px",
      "paddingBottom": "20px",
      "paddingTop": "20px",
    }
  }
>
  <div
    className="footerLinks"
    style={
      {
        "padding": "6px",
        "textAlign": "center",
      }
    }
  >
    <a
      href="https://www.facebook.com/pages/Apperson-Automotive/361511727202422"
    >
      <i
        aria-hidden="true"
        className="fab fa-facebook-square fa-2x"
        style={
          {
            "color": "#45c9ff",
          }
        }
      />
    </a>
    <a
      href="https://www.yellowpages.com/salem-va/mip/apperson-automotive-482421530"
      style={
        {
          "marginLeft": "40px",
        }
      }
    >
      <img
        alt="yellowpages"
        src="https://insightmg.com/wp-content/uploads/2015/12/YP_Insight-Marketing-Group_Marketing-for-Medical-Practices.png"
        style={
          {
            "marginBottom": "15px",
            "verticalAlign": "middle",
          }
        }
        width="40px"
      />
    </a>
    <a
      href="https://www.yelp.com/biz/apperson-automotive-salem"
      style={
        {
          "marginLeft": "30px",
        }
      }
    >
      <img
        alt="yelp"
        src="https://dl.dropboxusercontent.com/s/pb5qn5wyps6k56t/yelp-logo-icon-7.jpg?dl=0"
        style={
          {
            "marginBottom": "15px",
            "verticalAlign": "middle",
          }
        }
        width="40px"
      />
    </a>
  </div>
  <div
    style={
      {
        "padding": "6px",
        "textAlign": "center",
      }
    }
  >
    <span
      className="nohover"
    >
      Powered by Â 
    </span>
    <a
      className="wjllc"
      href="https://www.web-jam.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      Web Jam LLC
    </a>
  </div>
</div>
`;
